templates/cloud/partners/index.html

```diff
@@ -155,10 +155,9 @@ <h2 class="muted-heading">A selection of our cloud partners</h2>
 
 {% block footer_extra %}
 <script>
-    YUI().use('event-base', function (Y) {
-        Y.on('domready', function () {
-            core.loadPartners('?programme__name=Certified%20Public%20Cloud&programme__name=Charm%20Partner%20Programme&programme__name=OpenStack&featured=true', '#cloud-partners');
-        });
-    });
+    partnerLogoClouds = [{
```

Comment:
Could you at least add `<script src="{% versioned_static 'js/partners.js' %}"></script>` below the `partnerLogoClouds` before we merge this branch? I'm not comfortable merging work into `master` which knowingly breaks something that's currently working - since `master` could be released at any time.